test-data/project/fields/applied-menu-visibilities/transformers.ts

```diff
@@ -0,0 +1,25 @@
+import type {
+  TMenuVisibilities,
+  TAppliedMenuVisibilitiesGraphql,
+} from './types';
+
+import { Transformer } from '@commercetools-test-data/core';
+
+const upperFirst = (value: string): string =>
+  value.charAt(0).toUpperCase() + value.slice(1);
+
+const transformers = {
+  graphql: Transformer<TMenuVisibilities, TAppliedMenuVisibilitiesGraphql>(
+    'graphql',
+    {
+      replaceFields: ({ fields }) =>
+        Object.entries(fields).map(([key, value]) => ({
+          __typename: 'AppliedMenuVisibilities',
```

Comment:
Can‘t this be an add field.

Reply:
If you use `replaceFields` you can't (and does not make sense) to use `addFields` or `removeFields`.